### DISCUSSION TITLE
First built-in policy not matching

### DISCUSSION THREAD

**Discussion by ryzam** [dotnet/yarp#1214] [Q&A] [answered]
Hi guys,

I try to use 'First' loadbalancing policy but unfortunately get this error

`No matching ILoadBalancingPolicy found for the load balancing policy 'First' set on the cluster 'cluster1'` . My understanding YARP ships by default the following policies
First, Random, PowerOfTwoChoices (default), RoundRobin and LeastRequests without implement via ILoadBalancingPolicy.

**Comment by Kahbazi** [comment, marked as answer]
Try `FirstAlphabetical`. The `First` has been renamed to `FirstAlphabetical`. The doc will be updated in https://github.com/microsoft/reverse-proxy/pull/1215.
